devtools/README

```diff
@@ -174,6 +174,10 @@ ______________________________
     Achievements lists.
 
 
+make_class.py
+-------------------
+    Tool that adds all the boilerplate for a new C++ class inside an engine
+
```

Comment:
Please add at least one usage example. Demonstrate creation of class files in a subdirectory for a specific engine.

Reply:
Good idea, done.